include/rtc/peerconnection.hpp

```diff
@@ -35,6 +35,11 @@ struct RTC_CPP_EXPORT DataChannelInit {
 	string protocol = "";
 };
 
+struct RTC_CPP_EXPORT RemoteFingerprint {
+	string value;
+	CertificateFingerprint::Algorithm algorithm;
+};
```

Comment:
`struct CertificateFingerprint` implements [exactly this](https://github.com/paullouisageneau/libdatachannel/blob/eec049f6ddb3d2937fca8fe21443a87ba40e5225/include/rtc/description.hpp#L31), you shouldn't create a new struct.

Reply:
Done!